src/Components/SmartComponents/SystemsExposedTable/SystemsExposedTable.js

```diff
@@ -96,6 +101,12 @@ const SystemsExposedTable = ({ intl, cveName, cveStatusDetails, filterRuleValues
         );
     };
 
+    const expandAll = (rawData, isAllExpanded) => {
+        rawData.data.forEach(i => {
+            dispatch((expandRow(i.id, !isAllExpanded)));
```

Comment:
I think naming the current item for what it is is better for code readability, like here using `row` instead of `i` is more readable